### PR TITLE
Create Blog “how-to-run-openai-whisper-in-command-line/index”

### PR DESCRIPTION
**Pull request by MichaelJolley**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/posts/how-to-run-openai-whisper-in-command-line/index.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+---
+title: How to Run OpenAI Whisper in the Command Line
```
Comment (aimieye) [inline content/blog/posts/how-to-run-openai-whisper-in-command-line/index.md:2]: How to Run OpenAI Whisper via Command-Line in Google Colab

could we incorporate google colab somewhere

File: content/blog/posts/how-to-run-openai-whisper-in-command-line/index.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+description: test
```
Comment (aimieye) [inline content/blog/posts/how-to-run-openai-whisper-in-command-line/index.md:3]: In this blog, learn how to run the OpenAI Whisper speech recognition tool via Command-Line in Google Colab. Load it from the repository and get started now!

**Review by aimieye — COMMENTED** [review]
seo review complete

File: content/blog/posts/how-to-run-openai-whisper-in-command-line/index.md (hunk 1 above)
Comment (MichaelJolley) [reply content/blog/posts/how-to-run-openai-whisper-in-command-line/index.md:2]: This technically doesn't describe out to do that. We'll have better luck with a post coming from Nick

**Pull request closed without merging**